frodo.js

```diff
@@ -1,133 +1,126 @@
-const oddNumbers = numbers.filter(num => num % 2 !== 0);
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.